spring-cloud-gcp-data-spanner/src/main/java/org/springframework/data/spanner/core/mapping/Column.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.data.spanner.core.mapping;
```

Comment:
I don't see that you have changed the package name.
We are in the `Spring Cloud GCP` project. So, the package must be `org.springframework.cloud.gcp.data.spanner`.

Not sure that we should go that fine-grained name for the `core.mapping`.

More over I don't understand why we can't use JPA `@Column` annotation or at least don't go the same name do not lead to the ambiguity.
 :shrug: